mlir/test/Dialect/Vector/canonicalize.mlir

```diff
@@ -3446,3 +3446,68 @@ func.func @fold_insert_constant_indices(%arg : vector<4x1xi32>) -> vector<4x1xi3
   %res = vector.insert %1, %arg[%0, %0] : i32 into vector<4x1xi32>
   return %res : vector<4x1xi32>
 }
+
+// -----
+
+// CHECK-LABEL: @fold_insert_use_chain_static_pos(
+//  CHECK-SAME:   %[[ARG:.*]]: vector<4xf32>,
+//  CHECK-SAME:   %[[VAL:.*]]: f32) -> vector<4xf32> {
+//       CHECK:   %[[RES:.*]] = vector.insert %[[VAL]], %[[ARG]] [0] : f32 into vector<4xf32>
```

Comment:
Please make sure LIT variables match MLIR variables: https://mlir.llvm.org/getting_started/TestingGuide/#after-step-1-introduce-consistent-variable-names

Please apply similar changes to other tests.